### ISSUE TITLE
Invoice duplicates?

### ISSUE THREAD

**Issue by svanteschubert** [svanteschubert/complex-business-cases#11]
### Further Procedure 

Resend and check on receipt of invoice

No reference in electronic context

No direct possibility to mark duplicates

Best practice recommended?

Process question / process documentation?

**Comment by phax** [comment]
In Austria an invoice duplicate must be flagged as such, e.g. by printing the term "Duplikat" onto the invoice

**Comment by AP-G** [comment]
In Germany, the concept of a "similar multi-piece" has existed since around 2011. This means that the mandatory VAT information is identical in both documents. In this case, the word "duplicate" no longer needs to be printed or the indicator no longer needs to be set in the electronic file.

Nevertheless, I currently have the following case that is relevant in this context. There is a CIUS with a validity date. For example, a certain version of the CIUS is only valid until 31 January. An invoice is sent on 15 January. On 10 March, the recipient informs you that he has not received the invoice and needs it again. A new CIUS version with a different specification identifier has been valid since 1 February. Technically, invoices with the old identifier are no longer accepted. In this case, it is a challenge to resend a duplicate of the January invoice electronically.

**Comment by phax** [comment]
@AP-G  this is not a task for the CEN TC. This is a German national issue with the validity dates of XRechnung versions

**Comment by AndreasPvd** [comment]
@phax For the special national situation I agree. But this is just an example that can be valid for any CIUS/Extension when a specification identifier changes.

**Comment by phax** [comment]
Agree. But the versioning of CIUS and/or Extensions is not part of the TC 434. It's the responsibilities of the publishers/creators of these specifications

**Comment by AP-G** [comment]
That is true. What about the EN itself? Isn't it likely that the specification identifier will be updated soon? But you might be right that it is more a question on the VAT legislation experts at EU level. On the other hand, for them, it might be too technical…
